index.html

```diff
@@ -63,6 +63,6 @@ <h3 id="statistics-header">
     <script src="js/popup.js"></script>
     <script src="js/statistics.js"></script>
     <script src="js/scripts.js"></script>
-    <script src="https://cdnjs.cloudflare.com/ajax/libs/tabletop.js/1.5.2/tabletop.min.js"></script>
+    <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/PapaParse/5.1.0/papaparse.min.js" async defer></script>
```

Comment:
You may want to remove async and defer from this so that it loads *before* the map code 100% of the time. async will make the scripts load in any order, and defer is redundant here since the scripts are already at the bottom of the file.